### ISSUE TITLE
Underfloor heating 12 channels REG DeviceID B004

### ISSUE THREAD

**Issue by BastianZim** [ruebox/openhab2-addons#25]
``` xml
<device shortSerialNumber="AYQ" nameId="FEFF" functionId="FEFC" iconId="FFEF" softwareVersion="0.1.4" deviceId="B004" domainAddress="9376" compilerVersion="00000000" buildNumber="00000000" isTp="true" serialNumber="ABB2A00D2651" commissioningState="ready" copyId="a" progress="100">
<channels>
<channel mask="FFFFFFFF" nameId="004F" i="ch0000" cid="ABB2030B">
<attribute name="displayName">Fußboden</attribute>
<attribute name="floor">00</attribute>
<attribute name="functionId">27</attribute>
<attribute name="offsetX">0.261577569165276</attribute>
<attribute name="offsetY">0.633354728452621</attribute>
<attribute name="room">05</attribute>
<attribute name="selectedIcon">7</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0050" i="ch0001" cid="ABB2030B">
<attribute name="displayName">Fußbodenheizung</attribute>
<attribute name="floor">00</attribute>
<attribute name="functionId">27</attribute>
<attribute name="offsetX">0.49142853267609127</attribute>
<attribute name="offsetY">0.7022357723577236</attribute>
<attribute name="room">03</attribute>
<attribute name="selectedIcon">7</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>34</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0000" full="false">
<value>34</value>
</dataPoint>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0051" i="ch0002" cid="ABB2030B">
<attribute name="displayName">Fußbodenheizung</attribute>
<attribute name="floor">00</attribute>
<attribute name="functionId">27</attribute>
<attribute name="offsetX">0.5372837396315586</attribute>
<attribute name="offsetY">0.4130157955109127</attribute>
<attribute name="room">01</attribute>
<attribute name="selectedIcon">7</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0000" full="false">
<value>0</value>
</dataPoint>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0052" i="ch0003" cid="ABB2030B">
<attribute name="displayName">Fußbodenheizung</attribute>
<attribute name="floor">00</attribute>
<attribute name="functionId">27</attribute>
<attribute name="offsetX">0.48728373963155863</attribute>
<attribute name="offsetY">0.3947619241381448</attribute>
<attribute name="room">02</attribute>
<attribute name="selectedIcon">7</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>26</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0000" full="false">
<value>26</value>
</dataPoint>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0053" i="ch0004" cid="ABB2030B">
<attribute name="displayName">Fußboden</attribute>
<attribute name="floor">00</attribute>
<attribute name="functionId">27</attribute>
<attribute name="offsetX">0.4854320384837963</attribute>
<attribute name="offsetY">0.6771929824561403</attribute>
<attribute name="room">07</attribute>
<attribute name="selectedIcon">7</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0000" full="false">
<value>0</value>
</dataPoint>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0054" i="ch0005" cid="ABB2030B">
<attribute name="displayName">Fußbodenheizung</attribute>
<attribute name="floor">00</attribute>
<attribute name="functionId">27</attribute>
<attribute name="offsetX">0.5004986940719324</attribute>
<attribute name="offsetY">0.33114757433615094</attribute>
<attribute name="room">04</attribute>
<attribute name="selectedIcon">7</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>23</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0000" full="false">
<value>23</value>
</dataPoint>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0055" i="ch0006" cid="ABB2030B">
<attribute name="displayName">Fußbodenheizung</attribute>
<attribute name="floor">00</attribute>
<attribute name="functionId">27</attribute>
<attribute name="offsetX">0.4879806958712064</attribute>
<attribute name="offsetY">0.7018034054281933</attribute>
<attribute name="room">06</attribute>
<attribute name="selectedIcon">7</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0000" full="false">
<value>0</value>
</dataPoint>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0056" i="ch0007" cid="ABB2030B">
<attribute name="functionId">27</attribute>
<attribute name="selectedIcon">0</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0057" i="ch0008" cid="ABB2030B">
<attribute name="functionId">27</attribute>
<attribute name="selectedIcon">0</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0058" i="ch0009" cid="ABB2030B">
<attribute name="functionId">27</attribute>
<attribute name="selectedIcon">0</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0059" i="ch000A" cid="ABB2030B">
<attribute name="functionId">27</attribute>
<attribute name="selectedIcon">0</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
<channel mask="FFFFFFFF" nameId="0124" i="ch000B" cid="ABB2030B">
<attribute name="functionId">27</attribute>
<attribute name="selectedIcon">0</attribute>
<inputs>
<dataPoint i="idp0000" full="false">
<value>0</value>
</dataPoint>
</inputs>
<outputs>
<dataPoint i="odp0002" full="false">
<value>0</value>
</dataPoint>
</outputs>
<parameters>
<parameter i="pm0000">
<value>1</value>
</parameter>
<parameter i="pm0001">
<value>0</value>
</parameter>
</parameters>
<scenes/>
</channel>
</channels>
</device>
```
_Also added in the wiki_

**Comment by ruebox** [comment]
@Lucurus m Can you control this device via WebUI? If yes, could you please send me the excerpt for the different actions of XMPP via Chrome-WebDeveloper. Thx. E.g. on/off, percentage etc.

**Comment by BastianZim** [comment]
@ruebox Let me see what I can do, I'm not home right now so I'll probably be only able to answer next week.

**Comment by BastianZim** [comment]
Closed due to #32
